poco/lua/defold-poco.lua

```diff
@@ -0,0 +1,187 @@
+
+local VERSION = require('poco.lua.POCO_SDK_VERSION')
+local ClientConnection = require('poco.lua.ClientConnection')
+
+local PocoManager = {}
+PocoManager.__index = PocoManager
+
+PocoManager.DEBUG = false
+PocoManager.VERSION = VERSION
+
+PocoManager.server_sock = nil
+PocoManager.all_socks = {}
+PocoManager.clients = {}
+
+-- rpc methods registration
+local dispatcher = {
+    GetSDKVersion = function() return VERSION end,
+    Dump = function(onlyVisibleNode)
+        if onlyVisibleNode == nil then
+            onlyVisibleNode = true
+        end
+
+        if poco_helper ~=nil then
+            local w, h = window.get_size()
+            return poco_helper.dump(onlyVisibleNode, w, h)
+        end
+        return "No poco_helper extension found!"
+    end,
+    Click = function(x, y)
+        local w, h = window.get_size()
+        poco_helper.click(x * w, y * h)
+        return {}
+    end,
+    LongClick = function(x, y, duration)
+        local w, h = window.get_size()
+        poco_helper.long_click(x * w, y * h, duration)
+        return {}
+    end,
+    Swipe = function(x1, y1, x2, y2, duration)
+        local w, h = window.get_size()
+        poco_helper.swipe(x1 * w, y1 * h, x2 * w, y2 * h, duration)
+        return {}
+    end,
+
+    -- Screenshot = function(width)
+    --     width = width or 720
+    --     return Screen:getScreen(width)
+    -- end,
+    GetScreenSize = function()
+        local w, h = window.get_size()
+        return {width = w, height = h}
+    end,
+    -- SetText = function(_instanceId, val)
+    --     local node = Dumper:getCachedNode(_instanceId)
+    --     if node ~= nil then
+    --         return node:setAttr('text', val)
+    --     end
+    --     return false
+    -- end,
```

Comment:
I need to investigate this as well.
It would probably help when testing input fields and logging into services etc.